### PR TITLE
Add verify_ssl to OpenStack credential type

### PR DESCRIPTION
**Pull request by saito-hideki**
##### SUMMARY

This PR adds "Disable SSL Verification" checkbox to the credential edit page with openstack credential type.
If users turned it on, AWX/Tower can access the Keystone API ignoring verification failure that occur when using self-signed certificates.

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

 - UI

##### AWX VERSION

```
- devel
```

##### ADDITIONAL INFORMATION

- This is related on the issue #941 (RFE: OpenStack Credential Expantion)
- Re-created for #3132

### CODE REVIEW

**Comment by saito-hideki** [comment]
@AlanCoding @ryanpetrello I have re-created this separate PR for #3132 
If it is possible, can you review this again ?

File: awx/main/models/credential/__init__.py
Comment (AlanCoding) [inline awx/main/models/credential/__init__.py]: @ryanpetrello this is a little different from other sources where we tend to have `verify_ssl`, what do you think?

**Review by kdelee — APPROVED** [review]
Works for me as is, though @AlanCoding 's comment regarding making it consistent with the other credential types sounds good, I could try it out again after the change :+1:

File: awx/main/models/credential/__init__.py
Comment (ryanpetrello) [reply awx/main/models/credential/__init__.py]: @saito-hideki I think it would be nice to have this be consistent w/ the other sources and reverse the logic.

Comment (saito-hideki) [reply awx/main/models/credential/__init__.py]: @AlanCoding @ryanpetrello Thank you very much for the review!
So I will modify it from `no_verify_ssl` to `verify_ssl` as the same with Tower credential type and set the default value to True.

**Comment by saito-hideki** [comment]
@AlanCoding @ryanpetrello @kdelee Thank you for the follow-up.
I have finished to modify to use verify_ssl instead of no_verify_ssl and finished testing without errors.
Can you please review this ? :)

File: awx/main/migrations/0061_v350_add_openstack_cedential_verify_ssl.py
Comment (AlanCoding) [inline awx/main/migrations/0061_v350_add_openstack_cedential_verify_ssl.py]: I don't think this step is necessary because the default is True, so there's no need to add the field to all the existing credentials.

Comment (saito-hideki) [reply awx/main/migrations/0061_v350_add_openstack_cedential_verify_ssl.py]: @AlanCoding Thank you for your advice. I have removed the migration logic for the existing openstack credentials.
Please let me know if it does not sound correct.

File: awx/main/migrations/_credentialtypes.py
Comment (AlanCoding) [inline awx/main/migrations/_credentialtypes.py:224]: @ryanpetrello will this migration continue to be necessary?

Comment (ryanpetrello) [reply awx/main/migrations/_credentialtypes.py:224]: Nope!  We won't need the `0061` in this diff either.  Let's hold off on merging this until we see where we land on https://github.com/ansible/awx/pull/3257/

**Comment by ryanpetrello** [comment]
replaced by https://github.com/ansible/awx/pull/3262

**Pull request closed without merging**
